draft-ietf-rtcweb-jsep.xml

```diff
@@ -3567,7 +3584,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
               <t>ICE ufrag and password values, which MUST comply with
               the size limits specified in
-              <xref target="RFC5245" />, Section 15.4.</t>
+              <xref target="RFC8445" />, Section 5.3.</t>
```

Comment:
These limits don't seem clear enough - is the max length specified somewhere?

Reply:
Ah, it's in ietf-mmusic-ice-sip-sdp 4.4.  I've fixed the ref.